File: android/ECellVSM/app/src/main/java/com/aasim/ecellvsm/Models/Pending.java
package com.aasim.ecellvsm.Models;

/**
 * Created by aasim on 27/12/16.
 */

public class Pending {
    String _id,userid,name,longname;
    int type,quantity;

    public String get_id() {
        return _id;
    }

    public String getUserid() {
        return userid;

    }

    public String getName() {
        return name;
    }

    public String getLongname() {
        return longname;
    }

    public int getType() {
        return type;
    }

    public int getQuantity() {
        return quantity;
    }

}

File: node/models/pending.js
/**
 * Created by nidhi on 18/12/13.
 */
function pending(){
    this.userid = null;
    this.name = null;
    this.longname = null;
    this.type = null;
    this.quantity = null;
}

exports.create =  function () {
    var instance = new pending();
    return instance;
}